examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -120,7 +121,15 @@ class ReportCommand : public ModelCommand, public chip::app::ReadClient::Callbac
         params.mEventPathParamsListSize     = 0;
         params.mpAttributePathParamsList    = attributePathParams;
         params.mAttributePathParamsListSize = 1;
-
+        if (dataVersion != chip::kUndefinedDataVersion)
+        {
+            chip::app::DataVersionFilterParams dataVersionFilterParams[1];
+            dataVersionFilterParams[0].mEndpointId  = endpointId;
+            dataVersionFilterParams[0].mClusterId   = clusterId;
+            dataVersionFilterParams[0].mDataVersion = dataVersion;
+            params.mpDataVersionFilterParamsList    = dataVersionFilterParams;
```

Comment:
This is storing a pointer to something that is about to go out of scope.  I can only assume this codepath  is not being exercised in our tests...

Reply:
fixed, thanks